src/Tools/CodeGenerator/Featurizers.yaml

```diff
@@ -1244,18 +1240,17 @@ featurizers:
           - uint16
           - uint32
           - uint64
+          - bool
           - float
           - double
-          - bool
-          - string

```

Comment:
Did you mean to remove string from here?

Reply:
Good catch - pending checkin.